time stamp
int32 link_quality_num
int32 link_quality_den
int32 signal_level
int32 noise_level
